weather-app/src/app/app.component.ts

```diff
@@ -7,15 +8,29 @@ import { WeatherService } from './services/weather.service';
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
+  
 export class AppComponent implements OnInit {
-  constructor(private weatherService: WeatherService) {
-  }
+  isDarkMode!: boolean;
 
   public backgroundClassName: string = '';
   public title: string = 'SoulTeam Weather';
   public currentWeatherData: Observable<Object> = new Observable();
 
+  constructor(public localStorageThemeService: LocalStorageThemeService, private weatherService: WeatherService) { }
+  
   ngOnInit(): void {
+    const currentTheme: string | null = this.localStorageThemeService.getCurrentTheme();
+    if (!currentTheme) {
+      this.isDarkMode = false;
+      this.localStorageThemeService.setTheme('light');
+    }
+    if (currentTheme === 'light') {
```

Comment:
I think this condition could be implemented in a shorter way:
`this.isDarkMode  =  !!currentTheme === 'dark'`

Reply:
done